frontend/src/components/bauraten/foerdermix/FoerdermixFormular.vue

```diff
@@ -76,14 +86,81 @@ export default class FoerdermixFormular extends Mixins(
 
   private sumOver100 = false;
 
+  private isFreie = false;
+
+  private freieEingabe = "Freie Eingabe";
+
   @Prop({ type: Boolean, default: false })
   private readonly isEditable!: boolean;
 
+  private selectedItem: FoerdermixStammModel = createFoerdermixStammDto();
+
+  private stammdaten: FoerdermixStammModel[] = [];
+
+  private groupedStammdaten: GroupedStammdaten = [];
+
+  mounted(): void {
+    this.setGroupedStammdatenList();
+  }
+
   get gesamtsumme(): number {
     const sum: number = addiereAnteile(this.foerdermix);
     this.sumOver100 = sum > 100;
     return sum;
   }
+
+  get isFreieEingabe(): boolean {
+    if (this.selectedItem.foerdermix.bezeichnung === this.freieEingabe && this.isEditable) {
```

Comment:
Das If-Statement ist implizit true.

Somit sollte `return this.selectedItem.foerdermix.bezeichnung === this.freieEingabe && this.isEditable`  bzw. `feieEingabe = this.selectedItem.foerdermix.bezeichnung === this.freieEingabe && this.isEditable`ausreichen.